File: programming files/sum_even_fibonacci.java
public class sum_even_fibonacci {
    public static void main(String[] args) {
        // initialize variables
        int a = 0, b = 1;
        int sumEven = 0;
      
        while (b < 4000000) {
            // determine if b is even
            if (b % 2 == 0) {
                // add b to sum if even
                sumEven += b;
            }
            // update values of a and b
            int temp = b;
            b = a + b;
            a = temp;
        }
        // display sum of even values
        System.out.println(sumEven);
    }
}

// answer should be 4613732

File: programming files/sum_even_fibonacci.py
# initialize variables
a, b = 0, 1
sum_even = 0

while b < 4000000:
    # determine if b even
    if b % 2 == 0:
        sum_even += b
    # set a = b and b = a +b 
    a, b = b, a+b
# print values
print(sum_even)

# answer should be 4613732
